libs/map/src/lib/components/baselayer/vl-map-base-layer.ts

```diff
@@ -149,26 +152,48 @@ export class VlMapBaseLayer extends BaseElementOfType(HTMLElement) {
     }
 
     _createBaseLayer() {
+        const hasBackgroundLayer = this.hasAttribute('background-layer');
```

Comment:
Zoals besproken, eventueel zou je dit wat generieker kunnen maken om andere achtergrond lagen ook te ondersteunen in de toekomst. (Zo niet moet _cartoweb_ steeds de default blijven om backwards compatible te zijn)

V.b. `data-vl-background-layer="cartoweb"`

Jouw voorstel om een WMTS URL is ook een optie, maar dan zou het `data-vl-background-layer-wmts` oid moeten zijn. Echter de vraag is of dat voor eendere zou werken als daar andere parameters voor nodig zijn.

Reply:
Na bespreking met het team kwamen we tot conclusie dat `context-layer` als naam, meer context gaf over wat de variabele.

Andere opties:
- globe-layer
- world-layer
- ambient-layer
- root-layer
- surrounding-layer
- second-base-layer

Reply:
we gaan standaard voorlopig werken met cartoweb als default

op een later moment kan eventueel uitgebreid worden met andere background layers die dan parametriseerbaar zijn

als `data-vl-background-layer` aanwezig is zal er dus standaard een cartoweb achtergrondlaag aanwezig zijn